lesson02/elb_asg/common_vars.tf

```diff
@@ -0,0 +1,28 @@
+variable "aws_region" {
+  default = "us-east-1"
+}
+
+variable "metadata" {
+  default = {
+    appname    = "sample-app"
+    appversion = "latest"
+  }
+}
+
+variable "env" {
+  description = "string for the enviroment, allowed values develop, uat, prod"
+  default     = "develop"
+}
+
+variable "tags" {
+  type = "map"
+
+  default = {
+    Name     = "sample-app"
+    owner    = "carlos.castro@wizeline.com"
```

Comment:
shall we use `devops@wizeline.com` instead?

Reply:
yes